sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/StatsbeatTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Monitor.OpenTelemetry.Exporter.Internals;
+using Azure.Monitor.OpenTelemetry.Exporter.Internals.ConnectionString;
+using Xunit;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Tests
+{
+    public class StatsbeatTests
+    {
+        [Theory]
+        [InlineData("francecentral")]
+        [InlineData("francesouth")]
+        [InlineData("northeurope")]
+        [InlineData("norwayeast")]
+        [InlineData("norwaywest")]
+        [InlineData("swedencentral")]
+        [InlineData("switzerlandnorth")]
+        [InlineData("switzerlandwest")]
+        [InlineData("uksouth")]
+        [InlineData("ukwest")]
+        [InlineData("westeurope")]
+        public void StatsbeatConnectionStringIsSetBasedOnCustomersConnectionStringEndpointInEU(string euEndpoint)
+        {
+            var customer_ConnectionString = $"InstrumentationKey=1aa11111-bbbb-1ccc-8ddd-eeeeffff3333;IngestionEndpoint=https://{euEndpoint}.in.applicationinsights.azure.com/";
+            var parsedConectionString = ConnectionStringParser.GetValues(customer_ConnectionString);
+
+            Statsbeat.SetStatsbeatConnectionString(parsedConectionString.IngestionEndpoint);
```

Comment:
would recommend doing an more e2e test for this to simulate what a end user would do. They simply create tracer/loger/meter builder with azmon exporter added. Lets just do that and validate statsbeat got initialized with right endpoint.

Reply:
We are not enabling the statsbeat feature for next release. I will add this e2e test once we enable it.